CMakeLists.txt

```diff
@@ -143,6 +143,15 @@ list(APPEND CMAKE_PREFIX_PATH ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_PREFIX}/ll
 option(ENABLE_HIP_WORKAROUNDS Off)
 set(MIOPEN_INSTALL_CXX_HEADERS Off CACHE BOOL "Install MIOpen's C++ header interface")
 
+# Build flags for backward compatibility of MIOPEN Compiler flags
+set( MIOPEN_COMPILER_FLAG_VERSION V1 CACHE STRING
```

Comment:
Please use "PATHS" instead of "FLAGS" everywhere.

Reply:
Thanks, Enhanced this in latest version